src/platform/forms/helpers.js

```diff
@@ -38,14 +38,16 @@ export function groupPagesIntoChapters(routes, prefix = '') {
   }));
 }
 
-export function isInProgress(pathName) {
+export function isInProgress(pathName, additionalSafePaths = []) {
```

Comment:
Not that this is part of the changes you're making, but this can be a seriously misleading function name... Maybe something like `isInProgressPath`, but even then I'm not sure. Point is, the purpose is to mark pages that should get the in-progress treatment, but can be interpreted to mean pages that are actually in progress. Just kind of an unfortunate name for it.

Reply:
Agreed. This was new to me and confused me for a minute. I'm changing it. And while refactoring I noticed with have a duplicate version of this function in the `forms-system` helpers that I will leave alone because it's only used once and there is a ticket to consolidate the `forms` and `forms-system` code.